src/api/types.ts

```diff
@@ -3,3 +3,11 @@ export type DocumentSummary = {
   key: string;
   snapshot: string;
 };
+
+export type ProjectSummary = {
+  id: string;
+  name: string;
+  publicKey: string;
+  secretKey: string;
+  createdAt: string;
```

Comment:
It would be better to keep the original data type here to display it according to the condition such as width in view.

```suggestion
  createdAt: Date;
```

Reply:
I wanted to keep the data type as `Date`, but I met this error. 
Would it be better to allow this non-serializable value?

<img width="600" alt="" src="https://user-images.githubusercontent.com/81357083/167529734-b6109d68-f6b1-4ce8-ad9e-4c0b8e6a866e.png">

> One of the core usage principles for Redux is that [you should not put non-serializable values in state or actions](https://redux.js.org/style-guide/#do-not-put-non-serializable-values-in-state-or-actions). 
>
> If you _do_ need to turnoff those warnings, you can customize the middleware by configuring it to ignore specific action types, or fields in actions and state
> - https://redux-toolkit.js.org/usage/usage-guide#working-with-non-serializable-data

Reply:
@chacha912 Thanks for your reply.

I think it would be better to keep the option of non-serializable and store dates as timestamps(`number or string`). I found that RTK maintainer also recommends avoiding using timestamps.

https://github.com/reduxjs/redux-toolkit/issues/456#issuecomment-605556658

Reply:
Thanks for the recommendation. I changed the type of `createdAt` to number.